x-pack/plugins/reporting/public/plugin.tsx

```diff
@@ -112,7 +112,7 @@ export class ReportingPublicPlugin implements Plugin<void, void> {
         defaultMessage: 'Manage your reports generated from Discover, Visualize, and Dashboard.',
       }),
       icon: 'reportingApp',
-      path: '/app/kibana#/management/kibana/reporting',
+      path: '/app/management/insightsAndAlerting/reporting',
```

Comment:
@alexwizp @cjcenizal  are you sure this should be backported into `7.8` ?
It seems the section is registered in kibana https://github.com/elastic/kibana/blob/c4492524714038e7e888bc2788fa741356aeef3b/x-pack/plugins/reporting/public/plugin.tsx#L120

Reply:
Oops I’m sorry, you’re right. The regression was introduced into 7.9, not 7.8. So the fix doesn’t need to be backported to 7.8. My mistake!